examples/repository_collaborator/README.md

```diff
@@ -0,0 +1,22 @@
+# Repository Collaborator
+
+This provides a template for managing [repository collaborators](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository).
+
+This example will also create a repository in the specified `owner` organization or a personal account. See https://www.terraform.io/docs/providers/github/index.html for details on configuring [`providers.tf`](./providers.tf) accordingly.
```

Comment:
similar comment made in #478 -- we might want to remove reference to repo's made in a personal account for now?

Reply:
```suggestion
This example will also create a repository in the specified `owner` organization. See https://www.terraform.io/docs/providers/github/index.html for details on configuring [`providers.tf`](./providers.tf) accordingly.
```